doc/nrf/libraries/others/event_manager.rst

```diff
@@ -19,6 +19,10 @@ For a usage example of the Event Manger module and event structure in the |NCS|,
 
 See the :ref:`event_manager_sample` sample for a simple example of how to use the Event Manager.
 
+.. note::
+	Event manager had been updated and does no longer use orphaned sections to keep information about events.
+	You can remove :kconfig:`CONFIG_LINKER_ORPHAN_SECTION_PLACE` from config file.
```

Comment:
Maybe move this part to release notes. It may be a bit confusing to new readers who never encountered previous implementation.